app/styles/_box.scss

```diff
@@ -67,10 +71,15 @@
     height: 100%;
     width: 100%;
     @include background(linear-gradient(to right, black 35%, rgba(0,0,0,.3), rgba(0,0,0,.0)));
+  }
 
-    @include mq($medium-only) {
-      @include background(linear-gradient(to right, black, rgba(0, 0, 0, 0.5)));
-    }
+  @include mq($middle-narrow-only) {
+      background-position: center 40%;
+      filter: blur(3px);
```

Comment:
@noslouch I tested on Pixel and Samsung phones and it works well. On IE the blur doesn't show up but since the blur is only on mobile views, I don't think it's an issue.